impl/src/test/groovy/io/jsonwebtoken/DeprecatedJwtsTest.groovy

```diff
@@ -43,7 +44,7 @@ class DeprecatedJwtsTest {
     }
 
     protected static String toJson(o) {
-        def serializer = new RuntimeClasspathSerializerLocator().getInstance()
+        def serializer = new Services().loadFirst(Serializer)
```

Comment:
Just curious - why are you instantiating this class?

Unless I'm missing something, it should be marked as static final and the constructor should be private.

Reply:
Ahh, we didn't have a private constructor on `Services`, I'll add one, and fix this copypasta